src/Stores/Article.js

```diff
@@ -96,11 +95,13 @@ class Article {
     });
   }
 
+  unsubscribe() {
+    unsubscribe.next();
+    unsubscribe.complete();
+    unsubscribe = new Subject();
```

Comment:
@liamgold I think its not necessary to create a new instance of Subject here. Subject should be instantiated in some sort of 'create' method of the store / vue component. That would mean removing:

`let unsubscribe = new Subject();`

from top and adding it to that create method and finally removing the 

`unsubscribe = new Subject();`

Reply:
I managed to improve the sdk so that the http requests are resolved only after the observable is subscribed to. If you can, please try updating the `kentico-cloud-delivery` sdk to this version: https://www.npmjs.com/package/kentico-cloud-delivery/v/5.0.0-beta.2 

However, if you wanna test the unsubscribe/takeUntil functionality, you won't be able to do so with the use of 'delay' operator, but rather by actually slowing down the request itself. You might be able to simule this using Chrome's dev console.

Reply:
@Enngage I've restructured the Article store slightly so there's a subscribe method called on `mount` of the component.  Does this seem more appropriate?

If so, I can continue with the rest of the components and stores.

Also - I've updated the sdk as requested.